api/database/migrations/2023_12_12_204200_create_orders_table.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('store_id');
+            $table->unsignedBigInteger('store_id');
+            $table->unsignedBigInteger('category_id');
```

Comment:
1. `store_id` is duplicate
2. missing `user_id`
3. let's add cascade
4. $table->unsignedBigInteger('store_id');
    $table->foreign('store_id')->references('id')->on('stores');
Reference: https://laravel.com/docs/10.x/migrations#foreign-key-constraints

Can be shorten as below:

```suggestion
            $table->foreignId('user_id')
                ->constrained()
                ->onUpdate('cascade')
                ->onDelete('cascade');
            $table->foreignId('store_id')
                ->constrained()
                ->onUpdate('cascade')
                ->onDelete('cascade');
            $table->foreignId('category_id')
                ->constrained()
                ->onUpdate('cascade')
                ->onDelete('cascade');
```